pyxem/generators/orientation_refinement_generator.py

```diff
@@ -0,0 +1,205 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017-2019 The pyXem developers
+#
+# This file is part of pyXem.
+#
+# pyXem is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# pyXem is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with pyXem.  If not, see <http://www.gnu.org/licenses/>.
+
+"""
+Generating subpixel resolution on diffraction vectors.
+"""
+
+import numpy as np
+
+from pyxem.signals.diffraction_vectors import DiffractionVectors
+from pyxem.utils.expt_utils import peaks_as_gvectors
+from pyxem.utils.subpixel_refinements_utils import _conventional_xc
+from pyxem.utils.subpixel_refinements_utils import get_experimental_square
+from pyxem.utils.subpixel_refinements_utils import get_simulated_disc
+
+import warnings
+
+
+class OrientationRefinementGenerator():
```

Comment:
Can you make sure that the SubpixelRefinementGenerator() class name follows the same convention on capitalization.